### ISSUE TITLE
expose glossary link on hover

### ISSUE THREAD

**Issue by cjerdonek** [python/cpython#59898]
BPO | [15693](https://bugs.python.org/issue15693)
--- | :---
Nosy | @birkenfeld, @ezio-melotti, @bitdancer, @cjerdonek

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2012-08-16.15:48:28.210>
labels = ['type-bug', '3.8', '3.9', '3.10', 'docs']
title = 'expose glossary link on hover'
updated_at = <Date 2020-11-07.00:28:52.660>
user = 'https://github.com/cjerdonek'
```

bugs.python.org fields:
```python
activity = <Date 2020-11-07.00:28:52.660>
actor = 'iritkatriel'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2012-08-16.15:48:28.210>
creator = 'chris.jerdonek'
dependencies = []
files = []
hgrepos = []
issue_num = 15693
keywords = []
message_count = 9.0
messages = ['168388', '168389', '168391', '168393', '168394', '168395', '168398', '169169', '261495']
nosy_count = 5.0
nosy_names = ['georg.brandl', 'ezio.melotti', 'r.david.murray', 'chris.jerdonek', 'docs@python']
pr_nums = []
priority = 'normal'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue15693'
versions = ['Python 3.8', 'Python 3.9', 'Python 3.10']
```

</p></details>

**Comment by CAM-Gerlach** [comment]
This is really an upstream Sphinx issue, and was added in sphinx-doc/sphinx#8752 for Sphinx 3.5.0, which CPython 3.11+ uses to build its docs. Therefore, this issue can be closed, unless we want to backport the latest Sphinx 3.x (or later, to fix the PEP links) to the Python 3.10 branch.

**Comment by CAM-Gerlach** [comment]
This is really an upstream Sphinx issue, and was finally added in sphinx-doc/sphinx#8752 for Sphinx 3.5.0, which CPython 3.11+ uses to build its docs. Therefore, this issue can be closed, unless we want to backport the latest Sphinx 3.x (or later, to fix the PEP links) to the Python 3.10 branch.

@JulienPalard ?

**Comment by iritkatriel** [comment]
3.10 is in security-fix only mode now, so we can close this.
